### PR TITLE
Add info about ModuleConcatenationPlugin to README.md

### LINKED ISSUE #115: Chunk trees become opaque with ModuleConcatenationPlugin in Webpack 3
#### Issue description
Webpack 3introduces the plugin `webpack.optimize.ModuleConcatenationPlugin` for bundle optimizations. When used in conjunction with `BundleAnalyzerPlugin`, some chunks do not show their dependencies. Where there would once be a tree, there instead is an opaque label "<filename> + <X> modules". When the module concatenation plugin is removed, the tree becomes traceable again as expected.

#### Technical info
* Webpack Bundle Analyzer version: 2.9.0
* Webpack version: 3.5.6

### PR DESCRIPTION
**Pull request by valscion**
This PR adds documentation for `ModuleConcatenationPlugin`. Closes #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,14 @@ as Uglify, then this value will reflect the minified size of your code.
 
 This is the size of running the file(s) through gzip compression.
 
+## Troubleshooting
+
+### I can't see all the depencies in a chunk
```
Comment (th0r) [inline README.md]: Dependencies?

Comment (valscion) [reply README.md]: Oh my 😅

**Comment by valscion** [comment]
Better now? 😅 I can't believe how I missed that typo 🙈

**Comment by th0r** [comment]
Cool! Thanks, man! 👍

**Pull request merged**